### PR TITLE
[C-842] Add scrypt and account sagas to mobile

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add scrypt & account sagas after pair with @dylanjeffers 

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Tested locally with uncommented code in mobile sagas. Started to get past the initial account fetch and onto some local storage stuff.

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/services/audius-backend-instance.ts [hunk 1]
```diff
@@ -17,8 +20,67 @@ declare global {
...
+const LIBS_INITTED_EVENT = 'LIBS_INITTED_EVENT'
+
+/**
+ * Wait for the `LIBS_INITTED_EVENT` or pass through if there
+ * already exists a mounted `window.audiusLibs` object.
+ */
+const waitForLibsInit = async () => {
+  // If libs is already defined, it has already loaded & initted
+  // so do nothing
+  // @ts-ignore
```
Comment (dylanjeffers) [inline packages/mobile/src/services/audius-backend-instance.ts:36]: i think you can remove the @ts-ignore artifacts, but nbd

**Review by dylanjeffers — CHANGES_REQUESTED** [review]
not sure why the web package-lock has so many changes, otherwise looks great

File: packages/mobile/src/services/audius-backend-instance.ts (hunk 1 above)
Comment (raymondjacobson) [reply packages/mobile/src/services/audius-backend-instance.ts]: 👍

File: packages/mobile/package.json [hunk 2]
```diff
@@ -103,6 +103,7 @@
     "react-native-reanimated": "1.13.4",
     "react-native-safe-area-context": "3.3.2",
     "react-native-screens": "3.13.0",
+    "react-native-scrypt": "^1.2.1",
```
Comment (sliptype) [inline packages/mobile/package.json:106]:
```suggestion
    "react-native-scrypt": "1.2.1",
```

**Review by sliptype — APPROVED** [review]
Amazing!

Apparently @rickyrombo was able to prevent the package-lock sha changes with these steps:
1. Nuke node modules in each package and at root
2. Clear the cache npm cache clear --force
3. Make sure git is clean and npm i at root

Not sure if that works for you

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-scrypt-account-sagas

**Review by dylanjeffers — APPROVED** [review]
great stuff

**Pull request merged**